flexmeasures/api/v3_0/tests/test_sensor_schedules.py

```diff
@@ -76,14 +76,7 @@ def test_trigger_schedule_with_invalid_flexmodel(
 @pytest.mark.parametrize(
     "message, asset_name",
     [
-        (message_for_trigger_schedule(deprecated_format_pre012=True), "Test battery"),
```

Comment:
The ``deprecated_format_pre012`` parameter and its logic should also be removed from ``message_for_trigger_schedule()``.